lib/Unicode/Char/General.hs

```diff
@@ -428,6 +431,29 @@ isSymbol c = case generalCategory c of
     OtherSymbol    -> True
     _              -> False
 
+-- | Returns 'True' for alphabetic Unicode characters (lower-case, upper-case
+-- and title-case letters, plus letters of caseless scripts and modifiers
+-- letters).
+--
+-- prop> isLetter c == Data.Char.isLetter c
+--
+-- @since 0.1.0
+{-# INLINE isLetter #-}
+{-# DEPRECATED isLetter "Use isAlphabetic instead. Note that the behavior of this function does not match base:Data.Char.isLetter. See Unicode.Char.General.Compat for behavior compatible with base:Data.Char." #-}
+isLetter :: Char -> Bool
+isLetter = P.isAlphabetic
+
+-- | Returns 'True' for any whitespace characters, and the control
+-- characters @\\t@, @\\n@, @\\r@, @\\f@, @\\v@.
+--
+-- prop> isSpace c == Data.Char.isSpace c
```

Comment:
This does not hold.

Reply:
Yes, This is a deprecated function. I just readded it unchanged and deprecated it instead of removing it. We can remove it in the next release.

Reply:
Ok to readd but I think the doc should be fixed. See my MR.